src/mixed-line-bar-chart/internal.tsx

```diff
@@ -204,28 +203,37 @@ export default function InternalMixedLineBarChart<T extends number | string | Da
   const reserveFilterSpace = !showChart && !isNoMatch && (!hideFilter || additionalFilters);
   const mergedRef = useMergeRefs(containerRef, __internalRootRef);
 
+  const filterItems: ChartFilterProps<MixedLineBarChartProps.ChartSeries<T>>['series'] = useMemo(
```

Comment:
nit: I'm not sure we really need a memo for a simple `map` function that usually processes 1-10 items

Reply:
Thanks, I copied it from the existing chart-filters.tsx actually and forgot to remove the original file. I'll remove the file and get rid of the unnecessary memo.